engine/server/engine/enclave_manager/enclave_manager.go

```diff
@@ -253,6 +253,27 @@ func (manager *EnclaveManager) GetEnclaves(
 	return enclaveMapKeyedWithUuidStr, nil
 }
 
+func (manager *EnclaveManager) GetEnclavesByUuid(
```

Comment:
Same question as above - I thought the outcome of our last call was adding a filters param on GetEnclaves, rather than adding more endpoints? (I could be totally misremembering here)

Reply:
discussed offline